Resources/public/js/views/ez-usermenuview.js

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-usermenuview', function (Y) {
+    'use strict';
+    /**
+     * Provides the User Menu View class
+     *
+     * @module ez-usermenuview
+     */
+    Y.namespace('eZ');
+
+    var CLASS_HIDDEN = 'is-user-menu-hidden',
+        CLASS_USER_PROFILE = 'ez-user-profile',
+        SELECTOR_USER_PROFILE = '.' + CLASS_USER_PROFILE;
+
+    /**
+     * The User Menu view
+     *
+     * @namespace eZ
+     * @class UserMenuView
+     * @constructor
+     * @extends Y.eZ.View
+     */
+    Y.eZ.UserMenuView = Y.Base.create('userMenuView', Y.eZ.View, [], {
+        initializer: function () {
+            this.containerTemplate = '<ul class="ez-view-usermenuview ' + CLASS_HIDDEN + '"/>';
+
+            this.on('displayedChange', this._uiToggleUserMenu);
+            this.on('activeChange', this._setActiveViews);
+            this.after('displayedChange', this._setClickOutsideUserMenuHandler);
+
+            /**
+             * The click outside user menu event handler
+             *
+             * @method _clickOutsideUserMenuHandler
```

Comment:
it's a property (`@property`) not a method and it's should be marked as protected.